platform/admin/ui/src/main/webapp/js/views/installer/Finish.view.js

```diff
@@ -31,17 +31,23 @@ define([
             this.navigationModel = options.navigationModel;
             this.listenTo(this.navigationModel,'next', this.next);
             this.listenTo(this.navigationModel,'previous', this.previous);
+            this.model = this.navigationModel;
         },
         onClose: function() {
             this.stopListening(this.navigationModel);
         },
         next: function() {
+            this.showLoading();
             //this is your hook to perform any validation you need to do before going to the next step
             this.navigationModel.nextStep();
         },
         previous: function() {
             //this is your hook to perform any teardown that must be done before going to the previous step
             this.navigationModel.previousStep();
+        },
+        showLoading: function(){
```

Comment:
At the point where this is displayed, there is no toggle off. The system is restarting.